source/SIL.AppBuilder.Portal.Frontend/src/ui/components/product-artifact/header.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { titleize } from 'inflected';
+import Down from '@material-ui/icons/ArrowDropDown';
+import Up from '@material-ui/icons/ArrowDropUp';
+
+
+import ProductIcon from '@ui/components/product-icon';
+import {
+  ProductDefinitionResource,
+  ProductArtifactResource,
+  attributesFor
+} from '@data';
+import { i18nProps } from '@lib/i18n';
+
+interface IOwnProps {
+  productDefinition: ProductDefinitionResource;
+  artifacts: ProductArtifactResource[];
+  onClick: () => void;
+  isCollapsed: boolean;
+}
+
+type IProps =
+  & IOwnProps
+  & i18nProps;
+
+function humanReadableName(productDefinition) {
```

Comment:
this can go into a helper function I see using it in other pages

Reply:
for now, it's specific to the page.
it's the eqiv of:
```ts
humanize(attributesFor(anything).name)
```
which, idk if it's _really_ used in that many places.... like, I don't think _that_ many names need humanizing